src/main/java/com/liferay/ide/idea/ui/actions/WatchGradleModuleAction.java

```diff
@@ -38,6 +52,73 @@ public WatchGradleModuleAction() {
 		super("Watch", "Run watch task", LiferayIcons.LIFERAY_ICON, "watch");
 	}
 
+	public void afterTask() {
```

Comment:
hey @jtydhr88  can you review the changes I made this code over on the IDE side and resend here?  Thanks!